sale_stock_restocking_fee_invoicing/__manifest__.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 ACSONE SA/NV
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Stock Restocking Fee Invoicing",
+    "summary": """
+        On demand charge restocking fee for accepting returned goods .""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/account-invoicing",
+    "depends": ["sale", "sale_stock", "stock", "stock_account"],
```

Comment:
Why is stock_account required?

Reply:
To put the settings inside the valuation_setting_container which comes with stock_account